src/time_bucket.c

```diff
@@ -415,7 +415,12 @@ ts_time_bucket_ng_date(PG_FUNCTION_ARGS)
 	{
 		ereport(ERROR,
 				(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
-				 errmsg("origin must be the first day of the month")));
+				 errmsg("origin \"%04d-%02d-%02d\" must be the first day of the month",
```

Comment:
I would recommend against displaying a date in this hard-coded way. Different regions of the world use different format to display year, month, and day, so this can be confusing. US, for instance, is mm-dd-yyyy.

Reply:
OK, then I remove the date from the message for now. @gayyappan @erimatnor  If you don't mind, I would prefer not to dive into localizing error messages at this stage of the game. I'm going to refactor the error messages when the feature will be more stable.

Reply:
I added a corresponding note to the commit message.